src/job/MomoJobBuilder.ts

```diff
@@ -9,17 +9,30 @@ export class MomoJobBuilder {
   }
 
   withInterval(interval: string): this {
-    this.momoJob.interval = interval;
+    if (this.momoJob.schedule !== undefined && isInterval(this.momoJob.schedule)) {
+      this.momoJob.schedule = { ...this.momoJob.schedule, interval };
+      return this;
+    }
+
+    this.momoJob.schedule = { interval, firstRunAfter: 0 };
     return this;
   }
 
-  withCronSchedule(cronSchedule: string): this {
-    this.momoJob.cronSchedule = cronSchedule;
+  withFirstRunAfter(firstRunAfter: number): this {
+    if (this.momoJob.schedule !== undefined && isCronSchedule(this.momoJob.schedule)) {
+      throw new Error('Error: Setting a firstRunAfter on a cron job');
```

Comment:
note to self: check if we can somehow use type system to avoid throwing errors?

Reply:
You did check and implement this. It's been cherry-picked. Thanks!